packages/now-next/src/index.ts

```diff
@@ -77,7 +76,7 @@ interface BuildParamsType extends BuildOptions {
 }
 
 export const version = 2;
-
+const htmlContentType = 'text/html; charset=utf-8'
```

Comment:
Did you disable the precommit hook? Some of your lines end with a semicolon and some do not 🤔

Reply:
I didn't disable it on purpose, does something special need to be done to make sure it's triggered? 

<img width="927" alt="Screen Shot 2019-12-05 at 14 11 51" src="https://user-images.githubusercontent.com/22380829/70270075-32d44b00-1769-11ea-93f6-f4916f7d41bf.png">

Reply:
It should just be `yarn install` from the root directory.

```
$ cat .git/hooks/pre-commit

#!/bin/sh
# husky

# Hook created by Husky
#   Version: 3.0.4
#   At: 11/20/2019, 1:44:53 PM
#   ...etc...
```

Reply:
Manually ran `prettier` since it isn't being triggered